core/src/main/java/nl/nn/adapterframework/jdbc/dbms/OracleToH2Translator.java

```diff
@@ -103,8 +133,17 @@ private static String getConvertedQueryAsString(String[] newSplit, boolean remov
 			newSplit = convertQueryDropSequence(split);
 		} else if (isCreateIndexQuery(split)) {
 			newSplit = convertQueryCreateIndex(split);
+		} else if (isAlterTableIbisStoreQuery(split)) {
```

Comment:
Het zou toch niet nodig moeten zijn om hier Ibis specifieke translaties te hebben, toch?

Reply:
betreft IbisStore tabel welke onderdeel is van IAF. Met Oracel2ToH2Translator is het mogelijk om database aan te maken met enkel het Oracle create-database script